diff --git a/node_modules/oref0/lib/determine-basal/determine-basal.js b/node_modules/oref0/lib/determine-basal/determine-basal.js
index a599fc8..af02729 100644
--- a/node_modules/oref0/lib/determine-basal/determine-basal.js
+++ b/node_modules/oref0/lib/determine-basal/determine-basal.js
@@ -474,7 +474,7 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
     // use autosens-adjusted sens to counteract autosens meal insulin dosing adjustments so that
     // autotuned CR is still in effect even when basals and ISF are being adjusted by TT or autosens
     // this avoids overdosing insulin for large meals when low temp targets are active
-    csf = sens / profile.carb_ratio;
+    var csf = sens / profile.carb_ratio;
     console.error("profile.sens:",profile.sens,"sens:",sens,"CSF:",csf);
 
     var maxCarbAbsorptionRate = 30; // g/h; maximum rate to assume carbs will absorb if no CI observed
